content/doc/developer/publishing/releasing-cd.adoc

```diff
@@ -17,58 +17,70 @@ GitHub Actions are used to rebuild the code and deploy it to Artifactory,
 without the need for maintainers to use personal credentials or local builds.
 Release notes are generated automatically according to pull request (PR) titles and some predefined labels.
 
-`maven-release-plugin` (MRP) is not used in this mode.
+IMPORTANT: `maven-release-plugin` (MRP) is not used in this mode.
 Rather than traditional version numbers like `1.23` or `2.3.4` which are chosen by the maintainer,
 a CD release will have a version of the form `123.vabcdef456789`:
 an integer component determined by the Git history depth,
 and an informational component with the Git commit hash.
 
-== Prerequisites
+== Checklist
 
-. The plugin must already have been link:../../plugin-development/incrementals[incrementalified]. A new plugin created from the link:https://github.com/jenkinsci/archetypes/[archetypes] will already have this setup.
-. First enable the continuous delivery flag in link:https://github.com/jenkins-infra/repository-permissions-updater/[repository permission updater] (RPU) for your plugin.
+. Verify that the plugin has already been link:../../plugin-development/incrementals[incrementalified].
+  `pom.xml` should contain `<version>${revision}${changelist}</version>` before proceeding with subsequent steps.
```

Comment:
I think that escape is needed so that asciidoc does not expect to treat `${revision}` as a variable to be substituted.
```suggestion
  `pom.xml` should contain `<version>$\{revision}$\{changelist}</version>` before proceeding with subsequent steps.
```